Cookbook/Technical-Documents/BackendFeatureHowTo.md

```diff
@@ -0,0 +1,204 @@
+
+Let's define the concrete class for this protocol:
+``` swift
+struct CitiesBusinessController: CitiesBusinessControllerProtocol {
+    private let accessible: AuthenticatedAccessible
+
+    init(accessible: AuthenticatedAccessible) {
+        self.accessible = accessible
+    }
+
+    func fetch() -> SignalProducer<CitiesResponse, CoreError> {
+        let resource = CitiesService.fetchCitiesResource()
+        return accessible.access(resource)
+    }
+}
+```
+
+As simple as that: we get a backend resource from `CitiesService` , access it with the provided `AuthenticatedAccessible` and return the result.
+
+Note that we didn't specify the second parameter (request) for `accessible.access(…)`. Earlier we defined the request type as `Void`, which allows us to use a single-parameter convenience method on `AuthenticatedAccessible`, it simply calls `access(resource, with: ())`.
+
+Let's see how we may use the business controller in a view model.
+
+### View model
+In the view model, we usually keep the results of a fetch in the `State`:
+``` swift
+struct State {
+    var cities: [City]
+}
+```
+
+The entities we're loading are usually supplied through an `Event`:
+``` swift
+enum Event {
+    case didLoad(CitiesResponse)
+}
+```
+
+When the view model gets an event, `reduce` is called and we can set the new value:
+``` swift
+static func reduce(state: State, event: Event) -> State {
+    switch event {
+    case let .didLoad(citiesResponse):
+        return state.with(set(\.cities, citiesResponse.cities))
+    }
+}
+```
+
+Now `state.cities` contains the values we fetched and parsed from the network!
+
+In order to trigger loading, we define a `Feedback` and transform a network response in order to conform to the signature of the effect:
+``` swift
+{ _ -> SignalProducer<Event, NoError> in
+    citiesBusinessController.fetch()   // SignalProducer<CitiesResponse, CoreError>
+        .map(Event.didLoad)            // SignalProducer<Event, CoreError>
+        .replaceError(Event.didFail)   // SignalProducer<Event, NoError>
+}
+```
+
+Note that `citiesBusinessController` is provided as a parameter to the feedback function and doesn't require `self`.
+
+### Feature structure
+The business controller, service and related data types should all be grouped in an `API` directory if they are all for the same single feature:
+```
+Features
+└── Cities
+    ├── Tests
+    │   ├── Fixture/
+    │   ├── Canned Responses/
+    │   ├── CitiesBusinessControllerTests.swift
+    │   ├── CitiesServiceTests.swift
+    │   ├── CitiesDecodingTests.swift
+    │   ├── CitiesExpectations.swift
+    │   ├── CitiesViewModelTests.swift
+    │   ├── CitiesRendererSnapshotTests.swift
+    ├── API
+    │   ├── CitiesBusinessController.swift
+    │   ├── CitiesService.swift
+    │   ├── CitiesResponse.swift
+    │   └── City.swift
+    ├── CitiesViewModel.swift
+    ├── CitiesRenderer.swift
+    ├── CitiesFlowController.swift
+    └── CitiesBuilder.swift
```

Comment:
You mention the `Builder`, but no description regarding it. Could you put a small reference to it and mention the proposal `Builders as functions` #62.